website/docs/commands/fmt.md

```diff
@@ -22,15 +22,15 @@ scala-cli fmt --check
 
 ### Scalafmt version and dialect
 
-Scala CLI `fmt` command supports passing the `scalafmt` **version** and **dialect** directly from the command line, using respectively the `--scalafmt-dialect` and `--scalafmt-version` options:
+Scala CLI `fmt` command supports passing the `scalafmt` **version** and **dialect** directly from the command line, using the `--scalafmt-dialect` and `--scalafmt-version` options respectively:
 ```
 scala-cli fmt --scalafmt-dialect scala3 --scalafmt-version 3.5.8
 ```
-You can skip passing either of those, which will make Scala CLI to infer a default value:
+You can skip passing either of those, which will make Scala CLI infer a default value:
 - If a `.scalafmt.conf` file is present in the workspace and it has the field defined, the value will be read from there, unless explicitly specified with Scala CLI options.
-- Otherwise, the default `scalafmt` **version** will be the latest one used by your Scala CLI version (so it is subject to change when updating Scala CLI). The default **dialect** will be inferred based on Scala version (defined explicitly by `-S` option, or default version if option would not be passed).
+- Otherwise, the default `scalafmt` **version** will be the latest one used by your Scala CLI version (so it is subject to change when updating Scala CLI). The default **dialect** will be inferred based on Scala version (defined explicitly by `-S` option, or default version if the option is not passed).
```

Comment:
NIT
```suggestion
- Otherwise, the default `scalafmt` **version** will be the latest one used by your Scala CLI version (so it is subject to change when updating Scala CLI). The default **dialect** will be inferred based on the Scala version (defined explicitly by `-S` option, or default version if the option is not passed).
```

Reply:
Changed accordingly